File: typescript/src/identity/demographic.ts
export type BlindedDemographic = {
  data: string;
  version: number;
};

export type Demographic = {
  firstName?: string | undefined;
  middleName?: string | undefined;
  lastName?: string | undefined;
  maidenName?: string | undefined;
  gender?: string | undefined;
  race?: string | undefined;
  homePhoneNumber?: string | undefined;
  cellPhoneNumber?: string | undefined;
  email?: string | undefined;
  dob?: string | undefined;
  street?: string | undefined;
  city?: string | undefined;
  state?: string | undefined;
  zipCode?: string | undefined;
  mrn?: string | undefined;
  hcid?: string | undefined;
  ssn?: string | undefined;
  medicaidID?: string | undefined;
};

File: dotnet/src/CareEvolution.Orchestrate/Identity/Demographic.cs
using System.Text.Json.Serialization;

namespace CareEvolution.Orchestrate.Identity;

public sealed class Demographic
{
    public string? FirstName { get; set; }

    public string? MiddleName { get; set; }

    public string? LastName { get; set; }

    public string? MaidenName { get; set; }

    public string? Gender { get; set; }

    public string? Race { get; set; }

    public string? HomePhoneNumber { get; set; }

    public string? CellPhoneNumber { get; set; }

    public string? Email { get; set; }

    public string? Dob { get; set; }

    public string? Street { get; set; }

    public string? City { get; set; }

    public string? State { get; set; }

    public string? ZipCode { get; set; }

    public string? Mrn { get; set; }

    public string? Hcid { get; set; }

    public string? Ssn { get; set; }

    [JsonPropertyName("medicaidID")]
    public string? MedicaidId { get; set; }
}
